third_party/bigframes_vendored/sklearn/impute/_base.py

```diff
@@ -0,0 +1,56 @@
+# Authors: Nicolas Tresegnie <nicolas.tresegnie@gmail.com>
+#          Sergey Feldman <sergeyfeldman@gmail.com>
+# License: BSD 3 clause
+
+from bigframes_vendored.sklearn.base import BaseEstimator, TransformerMixin
+
+from bigframes import constants
+
+
+class _BaseImputer(TransformerMixin, BaseEstimator):
+    """Base class for all imputers.
+
+    It adds automatically support for `add_indicator`.
```

Comment:
What does this mean? Looks like we don't do that.

Reply:
Deleted.